.github/workflows/changeset-snapshot.yml

```diff
@@ -109,6 +109,8 @@ jobs:
 
       - name: Create and publish snapshot release
         uses: actions/github-script@v6
+        env:
+          GITHUB_TOKEN: ${{ secrets.CHANGESETS_TOKEN }}
```

Comment:
Late to this PR – just just wanted to know for my own knowledge. Why do we need both this line and line 115?